viz/tap/api/grpc_server.go

```diff
@@ -50,6 +51,8 @@ var (
 )
 
 // Tap is deprecated, use TapByResource.
+// This API endpoint is marked as deprecated but it's still used.
+//nolint:staticcheck
```

Comment:
I'm curious what staticcheck expects us to do about this one?  The `Tap` method really is deprecated but we still have to define an implementation to satisfy the contract.  I can understand warning about calls to a deprecated method, but implementations of a deprecated method should be allowed.